index.html

```diff
@@ -256,6 +256,9 @@ <h1>
         <li data-lang="th" id="mesodiar.com" data-owner="mesodiar" data-feed="https://medium.com/feed/mils-blog">
           <a href="https://mesodiar.com">mesodiar.com</a>
         </li>
+        <li data-lang="th" id="bigdatarpg.com/" data-owner="bigdatarpg" data-feed="https://bigdatarpg.com/">
```

Comment:
```suggestion
        <li data-lang="th" id="bigdatarpg.com" data-owner="bigdatarpg" data-feed="https://bigdatarpg.com/feed/">
```

Reply:
![image](https://user-images.githubusercontent.com/26707450/155710595-2569a118-811f-43f0-bcdb-2e6c040e8734.png)

Added วงแหวนเว็บ logo already

Reply:
Fixed suggestion and commited the new onealready